[2022-07-13 09:35:05,201] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_project_dag-12.download_dataset_task scheduled__2022-07-12T00:00:00+00:00 [queued]>
[2022-07-13 09:35:05,225] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_project_dag-12.download_dataset_task scheduled__2022-07-12T00:00:00+00:00 [queued]>
[2022-07-13 09:35:05,225] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-07-13 09:35:05,226] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-07-13 09:35:05,226] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-07-13 09:35:05,239] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2022-07-12 00:00:00+00:00
[2022-07-13 09:35:05,246] {standard_task_runner.py:52} INFO - Started process 103 to run task
[2022-07-13 09:35:05,259] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_project_dag-12', 'download_dataset_task', 'scheduled__2022-07-12T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/upload_to_gcs.py', '--cfg-path', '/tmp/tmph9c047tz', '--error-file', '/tmp/tmpg6y3jlvt']
[2022-07-13 09:35:05,265] {standard_task_runner.py:77} INFO - Job 3: Subtask download_dataset_task
[2022-07-13 09:35:05,366] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_project_dag-12.download_dataset_task scheduled__2022-07-12T00:00:00+00:00 [running]> on host 1baccf4ebc57
[2022-07-13 09:35:05,450] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_project_dag-12
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-12T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-12T00:00:00+00:00
[2022-07-13 09:35:05,452] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-13 09:35:05,453] {subprocess.py:74} INFO - Running command: ['bash', '-c', '#!/usr/bin\npip install kaggle\n\nexport KAGGLE_USERNAME=yusufkamorudeen\nexport KAGGLE_KEY=f9e7d64641f620123ab335c78451fdf4\n\n\n\n\n\ndataset_id=mashlyn\ndata_name=online-retail-ii-uci\n\nkaggle datasets download -d $dataset_id/$data_name --unzip -p "/opt/***/$dataset_id/$dataset_name"\n\n\n\n\n\n#mv $data_name uk_housing.zip\n\n\n']
[2022-07-13 09:35:05,478] {subprocess.py:85} INFO - Output:
[2022-07-13 09:35:08,147] {subprocess.py:89} INFO - Defaulting to user installation because normal site-packages is not writeable
[2022-07-13 09:35:08,887] {subprocess.py:89} INFO - Collecting kaggle
[2022-07-13 09:35:09,023] {subprocess.py:89} INFO -   Downloading kaggle-1.5.12.tar.gz (58 kB)
[2022-07-13 09:35:09,066] {subprocess.py:89} INFO -   Preparing metadata (setup.py): started
[2022-07-13 09:35:09,732] {subprocess.py:89} INFO -   Preparing metadata (setup.py): finished with status 'done'
[2022-07-13 09:35:09,740] {subprocess.py:89} INFO - Requirement already satisfied: six>=1.10 in /home/***/.local/lib/python3.7/site-packages (from kaggle) (1.16.0)
[2022-07-13 09:35:09,741] {subprocess.py:89} INFO - Requirement already satisfied: certifi in /home/***/.local/lib/python3.7/site-packages (from kaggle) (2020.12.5)
[2022-07-13 09:35:09,742] {subprocess.py:89} INFO - Requirement already satisfied: python-dateutil in /home/***/.local/lib/python3.7/site-packages (from kaggle) (2.8.2)
[2022-07-13 09:35:09,742] {subprocess.py:89} INFO - Requirement already satisfied: requests in /home/***/.local/lib/python3.7/site-packages (from kaggle) (2.26.0)
[2022-07-13 09:35:10,099] {subprocess.py:89} INFO - Collecting tqdm
[2022-07-13 09:35:10,123] {subprocess.py:89} INFO -   Downloading tqdm-4.64.0-py2.py3-none-any.whl (78 kB)
[2022-07-13 09:35:10,142] {subprocess.py:89} INFO - Requirement already satisfied: python-slugify in /home/***/.local/lib/python3.7/site-packages (from kaggle) (4.0.1)
[2022-07-13 09:35:10,143] {subprocess.py:89} INFO - Requirement already satisfied: urllib3 in /home/***/.local/lib/python3.7/site-packages (from kaggle) (1.26.7)
[2022-07-13 09:35:10,160] {subprocess.py:89} INFO - Requirement already satisfied: text-unidecode>=1.3 in /home/***/.local/lib/python3.7/site-packages (from python-slugify->kaggle) (1.3)
[2022-07-13 09:35:10,194] {subprocess.py:89} INFO - Requirement already satisfied: idna<4,>=2.5 in /home/***/.local/lib/python3.7/site-packages (from requests->kaggle) (3.3)
[2022-07-13 09:35:10,198] {subprocess.py:89} INFO - Requirement already satisfied: charset-normalizer~=2.0.0 in /home/***/.local/lib/python3.7/site-packages (from requests->kaggle) (2.0.9)
[2022-07-13 09:35:10,266] {subprocess.py:89} INFO - Building wheels for collected packages: kaggle
[2022-07-13 09:35:10,268] {subprocess.py:89} INFO -   Building wheel for kaggle (setup.py): started
[2022-07-13 09:35:11,052] {subprocess.py:89} INFO -   Building wheel for kaggle (setup.py): finished with status 'done'
[2022-07-13 09:35:11,052] {subprocess.py:89} INFO -   Created wheel for kaggle: filename=kaggle-1.5.12-py3-none-any.whl size=73051 sha256=adcc107f92181f0b325421ff99fad72b5ff44083ac932ac505e3a2396e5bdda9
[2022-07-13 09:35:11,052] {subprocess.py:89} INFO -   Stored in directory: /home/***/.cache/pip/wheels/62/d6/58/5853130f941e75b2177d281eb7e44b4a98ed46dd155f556dc5
[2022-07-13 09:35:11,055] {subprocess.py:89} INFO - Successfully built kaggle
[2022-07-13 09:35:15,999] {subprocess.py:89} INFO - Installing collected packages: tqdm, kaggle
[2022-07-13 09:35:16,324] {subprocess.py:89} INFO - Successfully installed kaggle-1.5.12 tqdm-4.64.0
[2022-07-13 09:35:16,508] {subprocess.py:89} INFO - WARNING: You are using pip version 21.3.1; however, version 22.1.2 is available.
[2022-07-13 09:35:16,508] {subprocess.py:89} INFO - You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
[2022-07-13 09:35:17,858] {subprocess.py:89} INFO - Downloading online-retail-ii-uci.zip to /opt/***/mashlyn
[2022-07-13 09:35:21,585] {subprocess.py:89} INFO -   0%|          | 0.00/14.5M [00:00<?, ?B/s]  7%|▋         | 1.00M/14.5M [00:00<00:01, 7.80MB/s] 14%|█▍        | 2.00M/14.5M [00:00<00:01, 8.73MB/s] 21%|██        | 3.00M/14.5M [00:00<00:01, 8.06MB/s] 41%|████▏     | 6.00M/14.5M [00:00<00:00, 13.5MB/s] 55%|█████▌    | 8.00M/14.5M [00:00<00:00, 12.8MB/s] 69%|██████▉   | 10.0M/14.5M [00:00<00:00, 13.0MB/s] 83%|████████▎ | 12.0M/14.5M [00:01<00:00, 13.1MB/s] 96%|█████████▋| 14.0M/14.5M [00:01<00:00, 13.0MB/s]100%|██████████| 14.5M/14.5M [00:01<00:00, 12.4MB/s]
[2022-07-13 09:35:21,589] {subprocess.py:89} INFO - 
[2022-07-13 09:35:21,837] {subprocess.py:93} INFO - Command exited with return code 0
[2022-07-13 09:35:21,979] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_project_dag-12, task_id=download_dataset_task, execution_date=20220712T000000, start_date=20220713T093505, end_date=20220713T093521
[2022-07-13 09:35:22,075] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-07-13 09:35:22,274] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
